### PR TITLE
7056 update talk in my talks

### PR DESCRIPTION
**Pull request by TanyaGordiychuk**

### CODE REVIEW

File: conference/react-js/src/actions/getTalksLangLevels.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+import { langLevels } from '../constants/backend-url';
```
Comment (blyzniuk) [inline conference/react-js/src/actions/getTalksLangLevels.js]: Should we have this 5 same files? Why not create one file 'Talk' with  all actions?

File: conference/react-js/src/components/Talks/TalksWrapper/TalksWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      isUpdateTalk: false,
+      talk: null,
+    };
+  }
+
+  getTalkById = (talks, id) => (
+    talks.filter(talk => talk.id === id)[0]
```
Comment (blyzniuk) [inline conference/react-js/src/components/Talks/TalksWrapper/TalksWrapper.jsx]: Array.prototype.find ?

File: conference/react-js/src/components/UpdateTalk/UpdateTalk.jsx [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+      topic: '',
+      type: '',
+      lang: '',
+      level: '',
+      addon: '',
+    };
+  }
+
+  componentDidMount() {
+    getTopics().then(({ data }) => {
```
Comment (blyzniuk) [inline conference/react-js/src/components/UpdateTalk/UpdateTalk.jsx]: Should we use Promise.all and single setState here?

File: conference/react-js/src/components/UpdateTalk/update-talk.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.update-talk {
+	&_wrapper {
```
Comment (blyzniuk) [inline conference/react-js/src/components/UpdateTalk/update-talk.scss:2]: looks like too many spaces

File: conference/react-js/src/components/UpdateTalk/UpdateTalk.jsx [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+
+    const { close } = this.props;
+
+    return (
+      <div className="update-talk_wrapper">
+
+        <div className="update-talk__title">
+          <TextField
+            floatingLabelText="Title"
+            style={{ flexGrow: 1 }}
```
Comment (blyzniuk) [inline conference/react-js/src/components/UpdateTalk/UpdateTalk.jsx]: Can we do without this?

**Review by ScreamCat1 — APPROVED** [review]

**Review by SupraVlad — APPROVED** [review]

**Pull request merged**